packages/plugins/plugin-atproto/.storybook/preview-head.html

```
@@ -0,0 +1 @@
+../../../../tools/storybook/.storybook/preview-head.html
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)